sourcemash/frontend/static/js/routers/app_router.js

```diff
@@ -26,6 +27,15 @@ Sourcemash.Routers.AppRouter = Backbone.Router.extend({
         this._swapView(feedsView);
     },
 
+    browseFeeds: function() {
+        var browseView = new Sourcemash.Views.BrowseView({
+            collection: new Sourcemash.Collections.Feeds(),
```

Comment:
So when initializing the Feeds Collection you would add {all: true}. Then in the file for Feeds, you would pass options to the initialize method that would set this.all = options.all. Then the URL function in the Feeds collection would append '/all' if this.all (is true).